test/fhirtypes/ElementDefinition.fixFshQuantity.test.ts

```diff
@@ -55,6 +60,34 @@ describe('ElementDefinition', () => {
       expect(referenceRangeLow.patternQuantity).toBeUndefined();
     });
 
+    it('should fix a FshQuantity to a quantity specialization', () => {
+      const onsetX = condition.elements.find(e => e.id === 'Condition.onset[x]');
+      onsetX.type = [new ElementDefinitionType('Age')];
+      onsetX.fixValue(fshQuantityAge, false, fisher);
+      // @ts-ignore we don't elaborate over all possible pattern[x] in ElementDefinition
```

Comment:
I agree that we don't elaborate over all `pattern[x]`, but I think every other type that can be set by some syntax in FSH directly, so all primitive types, and then the select complex types with a special syntax like CodeableConcept, Quantity, or Ratio are explicitly on the ElementDefinition. So is there precedent that `patternAge` should be on the ElementDefinition?

Reply:
I think the difference is that for those other ones we explicitly reference them in the production code (e.g., we have `src` code -- not just `test` code -- that uses `patternQuantity` directly). But we do not have `src` code that uses `patternAge` directly -- we only directly reference it in `test` code...

I could be convinced to add it... it just becomes unused when you package up the module without tests.

Reply:
I'm not sure that is generally the case. We do have `src` code that uses `patternQuantity`, but I can't find any that uses `patternCodeableConcept`, `patternCoding`, or most of the `pattern` elements it seems. From what I am seeing, `patternQuantity` is kind of special in that it does actually get used outside of tests.

That being said, I am still willing to be convinced we don't need to add `patternAge`, but I'm not convinced just yet.

Reply:
Ah... Hmm... Maybe we used to reference them directly but it got refactored?  Or maybe we never did?  I don't know.  I'll just add `patternAge` and `fixedAge` so you stop bugging me. ;-)

Reply:
OK.  Prepare to be overwhelmed by the glory of `fixedAge` and `patternAge` in 68f24d8.